paas/client.go

```diff
@@ -50,6 +50,176 @@ type CarrierClient struct {
 	Log           logr.Logger
 }
 
+// Services gets all Carrier services in the targeted org
+func (c *CarrierClient) Services() error {
+	log := c.Log.WithName("Services").WithValues("Organization", c.config.Org)
+	log.Info("start")
+	defer log.Info("return")
+	details := log.V(1) // NOTE: Increment of level, not absolute.
+
+	c.ui.Note().
+		WithStringValue("Organization", c.config.Org).
+		Msg("Listing services")
+
+	details.Info("validate")
+	err := c.ensureGoodOrg(c.config.Org, "Unable to list applications.")
+	if err != nil {
+		return err
+	}
+
+	secrets, err := c.kubeClient.Kubectl.CoreV1().
+		Secrets("carrier-workloads").
+		List(context.Background(), metav1.ListOptions{})
+	// TODO: See if we can prefilter for service + org secrets
```

Comment:
I think using a label or annotation will allow easier filtering.

Reply:
Likely yes. Have to learn labeling ... There is also the info about the SvCat resources you added to the ticket in response to my thoughts ... Thanks for that too.